packages/alfa-rules/src/common/expectations/audio-or-video-text-alternative.ts

```diff
@@ -0,0 +1,99 @@
+import { Question as ActQuestion } from "@siteimprove/alfa-act";
```

Comment:
@kasperisager Do you have an opinion about that refactor?
At the very least, it shows that we'r doing the exact same test (but different applicability) on these cases.

(affected rules: 26, 29, 31)

I am not entirely convinced that this is good. Notably trying to keep both the audio and video tests together. Refactoring only the video code used in 2 rules makes a lot of sense, I'm less sure about adding the audio part in it (even though it **is** the same code…)

Reply:
If we refer to the combination of "audio" and "video" as "media", then I think this might work a little better.

Reply:
Yes!